apps/antalmanac/src/actions/AppStoreActions.ts

```diff
@@ -120,18 +120,26 @@ export const loadSchedule = async (userId: string, rememberMe: boolean) => {
                 const res = await trpc.users.getUserData.query({ userId });
                 const scheduleSaveState = res?.userData;
 
-                if (scheduleSaveState === undefined) {
+                if (scheduleSaveState == null) {
                     openSnackbar('error', `Couldn't find schedules for username "${userId}".`);
-                } else if (await AppStore.loadSchedule(scheduleSaveState)) {
-                    openSnackbar('success', `Schedule for username "${userId}" loaded.`);
-                } else {
+                    return;
+                }
+
+                const isWebsocAlive = await isWebsocWorking();
+
+                if (!isWebsocAlive) {
```

Comment:
This doesn't account for when the initial "ping" succeeds but one of the subsequent requests fail. It's not that likely, but the previous code addressed that. I'm not sure why you wanted to change it.

Reply:
https://github.com/icssc/AntAlmanac/pull/702/files#r1332386420